relay-server/src/endpoints/common.rs

```diff
@@ -431,3 +431,24 @@ pub fn create_text_event_id_response(id: Option<EventId>) -> HttpResponse {
         .content_type("text/plain")
         .body(format!("{}", id.0.to_hyphenated()))
 }
+
+/// A helper for creating Actix routes that are resilient against double-slashes
+///
+/// Write `normpath("api/store")` to create a route pattern that matches "/api/store/",
+/// "api//store", "api//store////", etc.
+pub fn normpath(route: &str) -> String {
+    // Apparently the leading slash needs to be explicit and cannot be part of a pattern
+    let mut pattern = "/".to_owned();
+
+    for (i, segment) in route.split('/').enumerate() {
+        if i != 0 {
+            pattern.push_str(&format!("{{multislash{}:/+}}", i));
+        } else {
+            pattern.push_str(&format!("{{multislash{}:/*}}", i));
+        }
+        pattern.push_str(segment);
+    }
+
+    pattern.push_str("{trailing_slash:/*}");
```

Comment:
@tonyo I'm a bit confused now, since I thought the slash was only optional on the minidump endpoint in LBs. Apparently, we also made it optional for store now, so is it the right action to make it always optional?

Reply:
The trailing slash is only optional for the minidump endpoint at the moment, yes. Let's avoid allowing it anywhere else, if possible.

Reply:
api/store and api/x/store also allow a trailing slash, multiple even. no idea about LBs but in relay they do

Reply:
@tonyo can you clarify on that? 

The origin of this is https://github.com/getsentry/relay/pull/332, but there is no context given. I don't remember why this was added and whether this is explicit behavior that was allowed in Python store.

Reply:
Fairly sure Django normalizes much more than we currently do in master but I don't know how the LB setup works to know if those requests hit the right machines. They could've hit web instead of store so they worked accidentally, maybe